content/id/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Memperluas Kubernetes API dengan Lapisan Agregasi
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+Lapisan agregasi memungkinkan kubernetes untuk diperluas dengan API tambahan, di luar dari apa yang ditawarkan oleh inti API kubernetes.
```

Comment:
```suggestion
Lapisan agregasi memungkinkan kubernetes untuk diperluas dengan API-API tambahan, selain dari yang ditawarkan oleh API-API inti kubernetes.
```

Let's keep the plural forms.
Also, `inti API kubernetes` roughly means "the core of Kubernetes APIs" which can be misleading.
We should make it `API-API inti kubernetes` instead.

Reply:
Addressed here https://github.com/kubernetes/website/pull/16903/commits/6474e5e14057355e42e529f07f419ffed2a5a3bd